nomad-cluster-security.md

```diff
@@ -0,0 +1,14 @@
+# Nomad Cluster Security
+
+1. What is the noamd cluster size?
+1. Is Nomad deployed single region or multi-region? What is the deployment strategy? How is the high availability and failover designed? How is leader and follower selected?
+1. Is Nomad integrated with consul? How the DNS resolutions works to connect to the consul cluster?
+1. How is Nomad to Consul connectivity? Is it over HTTP secured with TLS? And is there any consul token to encrypt the traffic?
+1. Is the replication happens between the Nomad cluster in different regions?
+1. Does Nomad cluster uses the ACL policies, Tokens and sentinal policies?
```

Comment:
Should this be broken up into three separate questions? Can the answer be yes to some but not all?

Reply:
As suggested , broken into multiple questions 
  Do we have bootstrap Nomad ACL systems? 
  Do we have different tokens generated for authentication to Nomad API's ?
  Do we have policies defined as per user specific to schedule the job to Nomad Cluster ?
  Do we have sentinel policies defined in nomad clusters ?